mod/wikirate/template/type/wikirate_topic/structure/browse_item.haml

```diff
@@ -1,5 +1,5 @@
 .browse-topic-item
-  .text-center.topic-header
+  .text-center.topic-header{:title => nest(card, view: :name)}
```

Comment:
yeah, I didn't add it because `topic-header` already has ellipsis properties and this class is been used in other places. I will have to fix them first before I make changes to this class. (it is on my todo list as a part of clean up tasks and I prefer to do it after the review )